assets/js/components/JoyrideTooltip.js

```diff
@@ -38,6 +44,23 @@ export default function JoyrideTooltip( {
 	className,
 	styles = {},
 } ) {
+	const checkIfTargetExists = () =>
+		!! global.document.querySelector( target );
+
+	const [ targetExists, setTargetExists ] = useState( checkIfTargetExists );
+	const [ , cancel ] = useTimeoutFn( () => {
```

Comment:
Thanks!

1. Looks like I mixed up timeout with interval so I think we can update this with `useInterval`.
2. The intention was that it would continue to poll the DOM until the element existed, so yes
3. It looks like the interval could even be skipped using a delay of `null` which should address your 3rd point too I think.

Reply:
Ah, cool - yeah good shout on `setInterval`. As requested I have gone ahead and made the change.